CONTRIBUTING.md

```diff
@@ -26,25 +26,25 @@ The build requires the following:
 
 ## Creating a new plugin
 
-Plugins are found in the top-level `library` or `contrib` directories, depending on their level of support by the Buf team.
+Plugins are found in the top-level `plugins` directory.
 
-To create a new plugin, add a new folder matching the last component of the plugin's name and its version (i.e. `mkdir -p library/plugin-name/vX.Y.Z`) and add a `buf.plugin.yaml` / `Dockerfile` / `.dockerignore` to the newly created directory.
-To verify the plugin builds properly, run `make PLUGINS="<plugin-name>"` to build the Docker image and `make test PLUGINS="<plugin-name>"` to verify code generation for the plugin using some basic APIs stored in `tests/testdata/images/`.
+To create a new plugin, add a new folder with the plugin's organization, name, and version (i.e. `mkdir -p plugins/<org>/<name>/<version>`) and add a `buf.plugin.yaml` / `Dockerfile` / `.dockerignore` to the newly created directory.
```

Comment:
minor thought: would we want to put something in the contribution guide about the community/other organizations distinction? or some disclaimer like "get approval from <someone> before adding new organization"

Reply:
Good point - we should probably add something here or to the .github/ISSUE_TEMPLATE/plugin-request-for-buf-schema-registry.md. I may add this in a follow up as the build takes a while to run on the PR branch (because it is rebuilding all plugins).